requirements.yml

```diff
@@ -4,7 +4,7 @@
   name: o2-priority.java
 
 - src: git@github.com:o2-priority/ansible-elasticsearch.git
-  version: master
+  version: reporting
```

Comment:
assume this will be changed back to `master` once https://github.com/o2-priority/ansible-elasticsearch/pull/1 is merged

Reply:
I have changed the version back to master assuming elasticsearch will be merged to master as well.